app/assets/stylesheets/modules/spotlight_overrides.scss

```diff
@@ -40,22 +67,45 @@
   }
 }
 
-// Reduce size of page titles, headings, etc.
+// Adjust size of page titles, headings, etc.
 .page-title,
 .page-header {
-  font-size: 1.875rem;
   color: $heading-font-color;
+  font-size: 2rem;
 
   small {
     font-size: 1.5rem;
   }
 }
 
+.blacklight-browse h1 {
+  font-size: 2rem;
+}
+
 .item-count {
   color: $cool-grey;
   font-size: 1.25rem;
 }
 
+.tab-pane small.form-text {
```

Comment:
Just thinking of the change I made in projectblacklight/spotlight#2465, I'm wondering if this should not be constrained to only `small.form-text` elements that are children of a `.tab-pane`?

Reply:
Right, I saw that change you made and you're right it probably makes sense to do here. I'll make the update now.